src/client/components/Speksit/Speksit.css

```diff
@@ -97,100 +81,64 @@
 	font-size: 16px;
 }
 
-/* STYLES FOR H.A.L.I */
-
-.content_hali {
-	background-color: #580d0d;
-	margin: 0px;
-	padding: 0px;
-	padding-bottom: 20px;
-	text-align: left;
-	color: white;
-}
 .font_hali_header {
 	font-family: "Arial";
 	font-size: 36px;
 }
+
 .font_hali {
 	font-family: "Arial";
 	font-size: 16px;
 }
 
-/* STYLES FOR MOBILE DEVICES */ 
 @media ( min-width: 600px ) {
-	.kuvakaruselli {
-
-	}
-	.content_alaammuohi {
-
-	}
-	.content_kruunu {
-
-	}
-	.content_bratva {
-
-	}
-	.content_hali {
-
-	}
+  
+	
 }
 
 @media ( min-width: 840px ) {
-	.kuvakaruselli {
-		padding: 30px ;
-	}
-	.content_alaammuohi {
-		padding-top: 30px;
-		padding-bottom: 30px;
-	}
-	.content_kruunu {
-		padding-top: 30px;
-		padding-bottom: 30px;
-	}
-	.content_bratva {
-		padding-top: 30px;
-		padding-bottom: 30px;
-	}
-	.content_hali {
-		padding-top: 30px;
-		padding-bottom: 30px;
-	}
+	.speksiContainer {
+    flex-direction: row;
```

Comment:
check indentation

Reply:
did some fixing, editor shows it correctly but some conflicts tab/2spaces exist.